tests/serial/multiCouplingFourSolvers/MultiCouplingFourSolvers1.cpp

```diff
@@ -0,0 +1,23 @@
+#ifndef PRECICE_NO_MPI
+
+#include <string>
+
+#include <fstream>
+#include "../tests/serial/multiCouplingFourSolvers/helpers.hpp"
```

Comment:
I don't like the The `..`.

Reply:
Why would 
```suggestion
#include "helpers.hpp"
```
not work?

Reply:
:see_no_evil: Of course it works. Fixed it.